docs/docs/governance/cross-chain-governance.md

```diff
@@ -0,0 +1,134 @@
+# Cross Chain Governance
+
+To simplify the maintenance and management of the protocol contracts, we use a cross-chain governance that allows for a DAO proposal on mainnet to reach contracts on other networks.
+
+# How it works
+
+To reach other chains, calls emitted from the mainnet DAO go though the [Connext bridge](https://www.connext.network/) and are executed on the other side of the bridge, after a period of cooldown.
+
+```
+(mainnet)                       (destination chain)    (cooldown period)
+DAO proposal >  Connext Bridge >   Safe multisig    >  wait for 7 days   >
+```
+
+The workflow is as follow
+
+1. a DAO proposal is created, containing 1 call per chain
+2. if the vote succeed, the DAO proposal is executed. All calls are send to the bridge
+3. each call cross the bridge seperately towards its destination on a specific chain
+4. the call is received on the destination chain by Unlock multisig
+5. once received, the call is held in the multisig for a period of X days during which it can be cancelled.
+6. once the cooldown period ends, the call is executed
+
+NB: The cooldown period is useful to prevent malicious or errored calls from being executed
+
+The Safes still own Unlock contracts and Proxy Admins on destination networks (no changes in governance here).
+
+# Supported Networks
+
+(as Oct 10, 2023)
+
+|                    | Unlock Protocol | Safe | Connext |
+| ------------------ | --------------- | ---- | ------- |
+| Ethereum Mainnet   | x               | x    | x       |
+| Gnosis             | x               | x    | x       |
+| Polygon            | x               | x    | x       |
+| Optimism           | x               | x    | x       |
+| Arbitrum One       | x               | x    | x       |
+| BNB Chain          | x               | x    | x       |
+| Avalanche          | x               | x    |         |
+| Base               | x               | x    |         |
+| Celo               | x               | x    |         |
+| Palm               | x               |      |         |
+| Polygon zkEVM      |                 | x    |         |
+| zkSync Era mainnet |                 | x    |         |
+| Aurora             |                 | x    |         |
+
+# How to setup the multisig
+
+On every receiving chain, we need a Gnosis Safe configured with two Zodiac plugins
+
+- one Connext bridge receiver
+- one Delay
+
+1. In Gnosis web UI, go to Apps > Zodiac
+2. add the Delay module (cooldown : 7 days, expiration: 90 days)
```

Comment:
Let's reduce the cooldown to 48hours? (since the total DAO process is already quite slow...)